libraries.json

```
@@ -1422,5 +1422,22 @@
     "created": "2022-01-14",
     "updated": "2022-01-14",
     "version": 2
+  },
+  {
+    "name": "IT Logos",
+    "description": "Some more IT logos.",
```

```suggestion
    "description": "Some IT logos: MySQL, PHP, Jenkins, Tomcat, Bower, npm, Red Hat",
```

adding to description until we support filtering by item names